src/outputProcessor/HtmlOutputProcessor.ts

```diff
@@ -141,16 +141,29 @@ export default class HtmlOutputProcessor extends BaseOutputProcessor {
 
   constructor(
     protected file: string,
-    protected options: PluginOptions
+    protected options: PluginOptions,
+    /**
+     * Style CSS, gets placed within the `<style>` tag of the html document.
+     * 
+     * @default
+        body { font-family: monospace; }
+        p { margin: 0; padding: 0; }
+        pre { display: inline; margin: 0; }
+        h2 { margin: 0; font-size: 1.2em; }
+     */
+    protected style: string = `body { font-family: monospace; }
```

Comment:
Is this configurable? Shouldn't this then be documented in the readme?

Reply:
Added to readme